include/affinity.h

```diff
@@ -83,8 +70,12 @@ class thread_pinner {
 	bool m_successfully_initialized = false;
 };
 
+// Performs both naming and pinning (if enabled for this thread) of the invoking thread
+// This should be the first thing called in any thread that is part of the Celerity runtime
+void name_and_pin_thread(const named_threads::thread_type t_type);
```

Comment:
```suggestion
void name_and_pin_this_thread(const named_threads::thread_type t_type);
```
For consistency with `pin_this_thread`?